lib/controllers/authors.js

```diff
@@ -0,0 +1,22 @@
+const { Router } = require('express');
+const Author = require('../models/Author');
+const { post } = require('./books');
+
+module.exports = Router()
+  .post('/:id', async (req, res, next) => {
+    try {
+      const author = await Author.insert(req.body);
+      res.json(author);
+    } catch (e) {
+      next(e);
+    }
+  })
```

Comment:
Julie had accidentally placed post /authors/:id, we will actually want this to be /authors in the future to follow rest conventions.